<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLogon</class>
 <widget class="QWidget" name="frmLogon">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DataMusik Logon</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frmLogon
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(53, 53, 53), stop:1 rgba(150, 150, 150));
   font: &quot;URW Gothic L&quot;;
}

#gpbLogon
{
	background-color: rgba(0, 0, 0, 50);
}

#btnLogar
{
	background-color: rgb(0, 85, 0);
}

#btnNovo
{
	background-color: rgb(0, 0, 200);
}</string>
  </property>
  <widget class="QPushButton" name="btnLogar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Logar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNovo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Novo Usu√°rio</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="gpbLogon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	font: 12pt &quot;URW Gothic L&quot;;;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="txtSenha">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Senha*</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Nome de Logon*</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ptbImagem">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>121</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Imagens/Others/Funcionario.png) 0 0 0 0 stretch stretch;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNome</tabstop>
  <tabstop>txtSenha</tabstop>
  <tabstop>btnNovo</tabstop>
  <tabstop>btnLogar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
